psycopg/psycopg/_transformer.py

```diff
@@ -11,6 +11,8 @@
 from . import abc
 from ._cmodule import _psycopg
 
+__all__ = ("Transformer",)
+
```

Comment:
Unneeded, I think? Mypy has no problem with this definition and this is an internal module which people won't use, especially for `import *`.

Does the lack of __all__ causes problem to other static checkers?